src/model/document/document-element.ts

```diff
@@ -78,5 +77,8 @@ variableManager: shallowReadonly(
  */
 export interface QuantumElementCreationOptions {
   position?: Vector2
-  resizeable?: boolean
+  resizable?: boolean
+  size?: Vector2
```

Comment:
Note to self: It might be worthwhile to accept a `Vector2Like`, which would be an interface with a x and y coordinate